src/services/EdFi.DataManagementService.Api.Tests.Unit/Content/ContentProviderTests.cs

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Reflection;
+using System.Text;
+using EdFi.DataManagementService.Api.Content;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+
+namespace EdFi.DataManagementService.Api.Tests.Unit.Content;
+
+public class ContentProviderTests
+{
+    private IAssemblyProvider? _assemblyProvider;
+    private ILogger<ContentProvider>? _logger;
+    private Assembly? _assembly;
+
+    [SetUp]
+    public void Setup()
+    {
+        _assemblyProvider = A.Fake<IAssemblyProvider>();
+
+        _assembly = A.Fake<Assembly>();
+
+        var resources = new string[] { "file1.json", "file2.xsd", "file3.xsd" };
+
+        A.CallTo(() => _assembly.GetManifestResourceNames()).Returns(resources);
+
+        A.CallTo(() => _assemblyProvider.GetAssemby(A<Type>.Ignored)).Returns(_assembly);
+        _logger = A.Fake<ILogger<ContentProvider>>();
+    }
+
+    [Test]
+    public void Returns_Expected_Json_Files()
+    {
+        // Arrange
+        var contentProvider = new ContentProvider(_logger!, _assemblyProvider!);
+
+        // Act
+        var response = contentProvider.Files("file", ".json");
+
+        // Assert
+        response.Should().NotBeNull();
+        response.Count().Should().Be(1);
+        response.First().Should().Be("file1.json");
+    }
+
+    [Test]
+    public void Returns_Expected_Xsd_Files()
+    {
+        // Arrange
+        var contentProvider = new ContentProvider(_logger!, _assemblyProvider!);
+
+        // Act
+        var response = contentProvider.Files("file", ".xsd");
+
+        // Assert
+        response.Should().NotBeNull();
+        response.Count().Should().Be(2);
+        response.Should().Contain("file2.xsd");
+    }
+
+    [Test]
+    public void Returns_Expected_Json_File_Content()
+    {
+        // Arrange
+        var expectedHost = "http://local:5000";
+        var content = """{"openapi":"3.0.1", "info":"descriptors","servers":[{"url":"HOST_URL/data/v3"}]}""";
```

Comment:
The DMS isn't using "/data/v3".

* http://localhost:5198/ed-fi/students  - that works
* http://localhost:5198/data/v3/ed-fi/students - does not work

Reply:
Will include the data/v3 on the path pattern